crates/flux-middle/src/rty/mod.rs

```diff
@@ -1280,7 +1285,7 @@ pub enum BaseTy {
     Tuple(List<Ty>),
     Array(Ty, Const),
     Never,
-    Closure(DefId, /* upvar_tys */ List<Ty>),
+    Closure(DefId, /* upvar_tys */ List<Ty>, GenericArgs),
```

Comment:
I thought we were going to store a `flux_rustc_bridge::ty::GenericArgs` so we don't have to worry about the refinements, i.e., we store the "shape".

Reply:
We need to convert back to `rustc` -- is there a `to_rustc` for the `flux_rustc_bridge::ty::GenericArgs` ?

Reply:
oh wait there is...